web/src/features/map/Map.tsx

```diff
@@ -340,14 +326,18 @@ export default function MapPage({ onMapLoad }: MapPageProps): ReactElement {
     }
   };
 
+  const initialViewState =
```

Comment:
This would only work after the call to backend has succeeded AFAIK (why it was previously using an effect depending on `isSuccess`). Will we in a real-world scenario actually have backend data before the map is loaded? 🤔